### ISSUE TITLE
High contrast theme - extensions+charts

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#7724]
## Problem statement

Users with specific vision requirements find the interface difficult to read and navigate because the existing theme lacks sufficient color contrast. "As a PatternFly user, I would like to have more accessible options for my vision requirements. (from [https://github.com/patternfly/patternfly-design/issues/1357](https://github.com/patternfly/patternfly-design/issues/1357))"

## Goal

Update charts styles and any usage documentation/suggestions/demos needed to support products' ability to utilize HC charts in their UIs.

## Relevant links

- [Figma file for all extensions](https://www.figma.com/design/wKcOq7IrzfL1gEK2mocd6r/High-Contrast-Component---Extension-Specs?node-id=11908-85484)

- [https://github.com/patternfly/patternfly-design/issues/1357](https://github.com/patternfly/patternfly-design/issues/1357) - Design epic for HC theme

- [https://github.com/patternfly/design-tokens/issues/98](https://github.com/patternfly/design-tokens/issues/98) - Design issue for charts high contrast styles

## Issues

- [x] [https://github.com/patternfly/patternfly-quickstarts/issues/409](https://github.com/patternfly/patternfly-quickstarts/issues/409)

- [x] [https://github.com/patternfly/react-catalog-view/issues/87](https://github.com/patternfly/react-catalog-view/issues/87)

- [x] [https://github.com/patternfly/react-topology/issues/298](https://github.com/patternfly/react-topology/issues/298)

- [ ] [https://github.com/patternfly/patternfly-react/issues/12053](https://github.com/patternfly/patternfly-react/issues/12053)

- [ ] [https://github.com/patternfly/patternfly/issues/7949](https://github.com/patternfly/patternfly/issues/7949)

### Chatbot (feel free to do in groups)

- [x] [https://github.com/patternfly/chatbot/issues/696](https://github.com/patternfly/chatbot/issues/696)

- [x] [https://github.com/patternfly/chatbot/issues/697](https://github.com/patternfly/chatbot/issues/697)

- [x] [https://github.com/patternfly/chatbot/issues/698](https://github.com/patternfly/chatbot/issues/698)

- [x] [https://github.com/patternfly/chatbot/issues/699](https://github.com/patternfly/chatbot/issues/699)

- [x] [https://github.com/patternfly/chatbot/issues/700](https://github.com/patternfly/chatbot/issues/700)

- [x] [https://github.com/patternfly/chatbot/issues/701](https://github.com/patternfly/chatbot/issues/701)

- [x] [https://github.com/patternfly/chatbot/issues/702](https://github.com/patternfly/chatbot/issues/702)

- [x] [https://github.com/patternfly/chatbot/issues/703](https://github.com/patternfly/chatbot/issues/703)

- [x] [https://github.com/patternfly/chatbot/issues/704](https://github.com/patternfly/chatbot/issues/704)

- [x] [https://github.com/patternfly/chatbot/issues/705](https://github.com/patternfly/chatbot/issues/705)

- [x] [https://github.com/patternfly/chatbot/issues/706](https://github.com/patternfly/chatbot/issues/706)

- [x] [https://github.com/patternfly/chatbot/issues/707](https://github.com/patternfly/chatbot/issues/707)

- [x] [https://github.com/patternfly/chatbot/issues/708](https://github.com/patternfly/chatbot/issues/708)

- [x] [https://github.com/patternfly/chatbot/issues/709](https://github.com/patternfly/chatbot/issues/709)

- [x] [https://github.com/patternfly/chatbot/issues/710](https://github.com/patternfly/chatbot/issues/710)

- [x] [https://github.com/patternfly/chatbot/issues/720](https://github.com/patternfly/chatbot/issues/720) - new addition since Lucia's work

- [x] [https://github.com/patternfly/chatbot/issues/721](https://github.com/patternfly/chatbot/issues/721) - new addition since Lucia's work

- [x] [https://github.com/patternfly/chatbot/issues/723](https://github.com/patternfly/chatbot/issues/723)

- [x] [https://github.com/patternfly/chatbot/issues/722](https://github.com/patternfly/chatbot/issues/722)

---

**Jira Issue:** [PF-2519](https://redhat.atlassian.net/browse/PF-2519)

**Comment by mcoker** [comment]
Weekly update:
* Issues created and linked in this epic for all remaining tasks, including charts which is still in design
* Met with Rebecca and Eric and had an overview/howto sesh for the chatbot updates. They're going to handle those issues. Looks like these are already in PR review - https://github.com/patternfly/chatbot/pull/711, https://github.com/patternfly/chatbot/pull/712, and https://github.com/patternfly/chatbot/pull/713
* Topology changes are in a PR for review - https://github.com/patternfly/react-topology/pull/300

**Comment by rebeccaalpert** [comment]
Weekly update:
* We closed several that Lucia was happy with in ChatBot - were getting PatternFly styles automagically - and have some new PRs up for review.
* React topology is in PR and approved by design - https://github.com/patternfly/react-topology/pull/300

**Comment by mcoker** [comment]
Weekly update:
* Merged catalog view - https://github.com/patternfly/react-catalog-view/pull/91
* Charts HC is still in progress

**Comment by mcoker** [comment]
All work is complete
